app/README.md

```diff
@@ -93,22 +93,16 @@ Here is a full breakdown of the available options for the JavaScript library:
 </script>
 ```
 
-### Handling Zendesk community search
-
-We do not index community forums at the moment. If you're using them, you'll probably want to disable `instantsearch` by setting `enabled: false`.
-
 ### Customizing the CSS
 
-If you want to change the CSS styling of the search we provide, there's absolutely no issue.
-To do this:
+If you want to customize the look & feel of the search, you can follow the following steps:
+
+- Ensure you read the [SCSS source file](https://github.com/algolia/algoliasearch-zendesk/blob/master/app/css/index.scss) to see all the rules we're using
+- Add a `<style>` tag after the `<link>` tag you've already added in your **Document Head** template
+  We ask you to do this because the CSS code in the customization panel is included __before__ the **Document Head** template
 
-- You should have a look at the [scss file](https://github.com/algolia/algoliasearch-zendesk/blob/master/app/css/index.scss) to see all the rules we're using
-- Add a `<style>` tag after the `<link>` tag you've already added in your *Document Head* template
-  We ask you to do this because the CSS code in the customization panel is included __before__ the *Document Head* template
-- Fix the versions of the JavaScript and CSS files by replacing the `/1/` in the URLs by `/1.X/` or `/1.X.Y/`. (Current version: [![tag](https://img.shields.io/github/tag/algolia/algoliasearch-zendesk.svg)](https://github.com/algolia/algoliasearch-zendesk/releases))
-  Indeed, we might do some small CSS changes between minor versions.
 
```

Comment:
I feel like not mentioning this will backfire if we want to do any markup/style update that breaks the style of one of our customers.